src/veh_interact.cpp

```diff
@@ -1165,6 +1167,7 @@ void veh_interact::do_tirechange()
     display_mode('c');
     werase( w_msg );
     int msg_width = getmaxx(w_msg);
+    bool has_str = (g->u.get_str() * TIRE_CHANGE_STR_MOD) >= veh->total_mass();
```

Comment:
How about this: splitting the value into
```
const int needed_strength = veh->total_mass() / TIRE_CHANGE_STR_MOD;
const bool has_str = g->u.get_str() >= needed_strength;
```
And then below, in the print message you could say how much strength is required to replace the jack.